source/Stargate-Model-Tests/QualifiedLanguageRangeTest.class.st

```diff
@@ -0,0 +1,91 @@
+"
+A QualifiedLanguageRangeTest is a test class for testing the behavior of QualifiedLanguageRange
+"
+Class {
+	#name : #QualifiedLanguageRangeTest,
+	#superclass : #TestCase,
+	#category : #'Stargate-Model-Tests-Controllers'
+}
+
+{ #category : #tests }
+QualifiedLanguageRangeTest >> testCantParseInvalidStrings [
+
+	self
+		should: [ QualifiedLanguageRange fromString: '' ]
+			raise: InstanceCreationFailed
+			withMessageText: 'Cannot create a qualified language range with an empty string';
+		should: [ QualifiedLanguageRange fromString: 'en;q=' ] raise: Error;
```

Comment:
Why are we not checking for the message text?

Reply:
Because this error in particular is not generated by our code and later is handled and ignored. :wink: